Bacpac/Backup-NavContainerDatabases.ps1

```diff
@@ -68,7 +68,12 @@ try {
             $params = @{}
             if ($compress) { $params += @{ "CompressionOption" = "On" } }
             if ($databaseCredential) { $params += @{ "credential" = $databaseCredential } }
-            Backup-SqlDatabase -ServerInstance $serverInstance -database $database -BackupFile $bakFile @params
+

```

Comment:
Text says importing?
Is this really a problem with backup? Database size in SQL Express is limited - did you ever run into this?